### DISCUSSION TITLE
Mac OS support?

### DISCUSSION THREAD

**Discussion by akparhi** [Dokploy/dokploy#1044] [Q&A]
I'd like to run dokploy on a mac OS vm, trying to install the script on mac does not seem to work.

**Comment by Siumauricio** [comment]
No, dokploy is supported only in these distributions https://docs.dokploy.com/docs/core/installation#virtual-private-server-vps

**Reply by akparhi** [reply]
Any plans to support mac os? Mac VPSes have been useful for a bunch of things for us.
